src/main/java/cloud/cholewa/game/Player.java

```diff
@@ -0,0 +1,26 @@
+package cloud.cholewa.game;
+
+import cloud.cholewa.config.ErrorDict;
+
+public class Player {
+    private String name;
+
+    private Player() {
+    }
+
+    public Player(String name) {
+        if (name == null) {
+            throw new IllegalArgumentException(ErrorDict.PLAYER_CREATE_ERROR_CANT_BE_NULL);
+        }
+        if (name.equals("")) {
+            throw new IllegalArgumentException(ErrorDict.PLAYER_CREATE_ERROR_CANT_BE_EMPTY_STRING);
+        }
+
```

Comment:
Tu jest generalnie ok, tylko możnaby się zastanowić czy jest sens rozbijać na dwa różne przypadki, które sprowadzają się od strony użytkownika to jednego "puste imię", może lepiej podpiąć to pod jeden tekst i ściąć nadmiernego ifa.

Reply:
Poprawione do jednego przypadku... chociaż na ten moment pozbawiło to sensu, trzymanie osobnej klasy słownika. :)